<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUserComponent</class>
 <widget class="QTabWidget" name="CUserComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tb_AllUsers">
   <attribute name="title">
    <string>Users</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_AllUsers">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="BlackGui::Views::CUserView" name="tvp_AllUsers">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_Clients">
   <attribute name="title">
    <string>Clients</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_Clients">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="BlackGui::Views::CClientView" name="tvp_Clients">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Views::CUserView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/userview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CClientView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/clientview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
